docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,498 @@
+{
```

Comment:
<p>I think you should replace the window with a vertical line in the matrix profile figure. For example, see the section  <code>Find potential anomalies (discords) using stump</code> in this page: https://stumpy.readthedocs.io/en/latest/Tutorial_STUMPY_Basics.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>